### PR TITLE
Albany

### PR DESCRIPTION
**Pull request by bsuvorov**
Source code changes to enable Albany parsing. 
What's left: parse out event timespace and professor name.

let it be the slowest Scala parser I wrote  - one commit per month and more commits coming.

### CODE REVIEW

File: backend/src/main/scala/course_refresh/package-info.java [hunk 1]
```diff
@@ -0,0 +1,4 @@
+/**
+ * Created by bsuvorov on 12/21/14.
+ */
+package course_refresh;
```
Comment (vovagalchenko) [inline backend/src/main/scala/course_refresh/package-info.java]: What's this file for?

File: backend/src/main/scala/fetch_managers/AlbanyCourseFetchManager.scala [hunk 2]
```diff
@@ -0,0 +1,261 @@
...
+import com.typesafe.scalalogging.slf4j.LazyLogging
+import course_refresh.{HTTPRequest, HTTPManager, SchoolManager}
+import model._
+
+import scala.concurrent.Future
+import scala.xml.{Node, NodeSeq}
+import scala.concurrent.ExecutionContext.Implicits.global
+
+class AlbanyCourseFetchManager(term: String) extends SchoolManager with LazyLogging {
+  lazy val allDepartments: Future[NodeSeq] = {
```
Comment (vovagalchenko) [inline backend/src/main/scala/fetch_managers/AlbanyCourseFetchManager.scala]: I think you only use this inside fetchDepartments – you might as well just do this there.

File: backend/src/main/scala/fetch_managers/AlbanyCourseFetchManager.scala [hunk 3]
```diff
@@ -0,0 +1,261 @@
...
+          val headText = schoolSpecificId.head.text
+          if (headText.length > 0) {
+            val schoolSpecificId : String = name.replaceAll(" ","_")
+            Some(Department(SchoolId.Albany, schoolSpecificId, name))
+          } else {
+            None
+          }
+        }
+      }
+      test
```
Comment (vovagalchenko) [inline backend/src/main/scala/fetch_managers/AlbanyCourseFetchManager.scala]: you can just remove this variable.

File: backend/src/main/scala/fetch_managers/AlbanyCourseFetchManager.scala [hunk 4]
```diff
@@ -0,0 +1,261 @@
...
+//          println(sectionType)
+//          println(seatsRemaining)
+
+          (section, event :: Nil)
+      }
+
+      sectionAndEvent.toSeq
+    }
+
+    t
```
Comment (vovagalchenko) [inline backend/src/main/scala/fetch_managers/AlbanyCourseFetchManager.scala]: You can get rid of this variable.

File: backend/src/main/scala/course_refresh/package-info.java (hunk 1 above)
Comment (bsuvorov) [inline backend/src/main/scala/course_refresh/package-info.java]: removed this file. must have added by mistake.

File: backend/src/main/scala/fetch_managers/AlbanyCourseFetchManager.scala (hunk 2 above)
Comment (bsuvorov) [inline backend/src/main/scala/fetch_managers/AlbanyCourseFetchManager.scala]: done.

**Pull request merged**